WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2301,8 +2268,8 @@ private String migrateLegacyDraft(String content) {
             while (matcher.find()) {
                 String stringUri = matcher.group(1);
                 Uri uri = Uri.parse(stringUri);
-                MediaFile mediaFile = FluxCUtils.mediaFileFromMediaModel(
-                        queueFileForUpload(uri, getContentResolver().getType(uri), MediaUploadState.FAILED));
+                MediaFile mediaFile = FluxCUtils.mediaFileFromMediaModel(mEditorMedia
```

Comment:
Please try to carefully compare the code in develop against the code in this branch as we can't easily test this.

Imho we invoke `queueFileForUpload` but the upload isn't started as MediaUploadState is set to FAILED and upload is started only for items with MediaUploadState.QUEUED.